modules/network-observability-metrics.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-metrics_{context}"]
+= Network Observability metrics
+Metrics generated by the `flowlogs-pipeline` are configurable in the `spec.processor.metrics.includeList` of the `FlowCollector` custom resource to add or remove metrics. 
+
+You can also create alerts by using the `includeList` metrics in Prometheus rules, as shown in the example "Creating an alert for high ingress traffic".
+
+When looking for these metrics in Prometheus, such as in the Console through Observe -> Metrics, or when defining alerts, all the metrics names are prefixed with "netobserv_". For example, "netobserv_namespace_flows_total". Available metrics names are as follows. Names followed by `*` are enabled by default: 
```

Comment:
`Names followed by `*` are enabled by default:` line can be removed here. since its mentioned in L16 below